### ISSUE TITLE
Only Automatically Close Posting Window if Pasting Successful

### ISSUE THREAD

**Issue by smcgregor** [privly/privly-chrome#113]
Currently the posting window closes automatically when posting is initiated from the posting button or the context menu **and:**

* The host page's tab closes
* (or) Posting completes

We should change this to only close automatically in the event that the posting completes and the link is successfully pasted into the host page's document.

**Comment by SummerWish** [comment]
Agree. But if the action is always unsuccessful (like the Youtube case without a hacking patch), what should we do? Alert users something like *inserting Privly link failed, please manually insert it*?

**Comment by smcgregor** [comment]
We already have prompts telling them to copy/paste the link. They only see the prompts if the link is not auto-posting, so I think this is already good.

**Comment by smcgregor** [comment]
Downgraded to minor because this is less of an issue for Seamless.
